flow-typed/npm/jest_v26.x.x.js

```diff
@@ -942,22 +942,22 @@ type JestDoneFn = {|
 
 /** Runs this function after every test inside this context */
 declare function afterEach(
-  fn: (done: JestDoneFn) => Promise | null<mixed>,
+  fn: (done: JestDoneFn) => ?Promise<mixed>,
   timeout?: number
 ): void;
 /** Runs this function before every test inside this context */
 declare function beforeEach(
-  fn: (done: JestDoneFn) => Promise | null<mixed>,
+  fn: (done: JestDoneFn) => ?Promise<mixed>,
```

Comment:
were these changes manually applied? we haven't write this ourselves, this comes from the flow-typed public repository, and in theory we shouldn't need to make any change to this since it's JS/flow